addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/LoxoneHandlerFactory.java

```diff
@@ -24,8 +29,12 @@
  *
  * @author Pawel Pieczul - Initial contribution
  */
+@NonNullByDefault
+@Component(service = ThingHandlerFactory.class, immediate = true)
 public class LoxoneHandlerFactory extends BaseThingHandlerFactory {
 
+    @SuppressWarnings("null")
```

Comment:
Maybe you could consider leaving this ones out

Reply:
Removing the `@SuppressWarnings` here will give a compiler warning....

Reply:
`Sets` comes from com.google.common.collect package and we talked about not suppressing only JDK calls, that's why I left it. However, I am ok with removing this and leaving the warning.

Reply:
That is a valid point 👍

Reply:
what about dropping the unnecessary dependency to the google library and just use
```
public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = new HashSet<>(LoxoneBindingConstants.THING_TYPE_MINISERVER);
```

Reply:
Probably better to use: `Collections.singleton()`. Which is also standard java.